insanely_fast_whisper_rocm/core/integrations/diarization.py

```
@@ -0,0 +1,345 @@
+            # to avoid leaked device memory.
+            try:
+                import torch
+
+                pipeline.to(torch.device("cpu"))
+                if torch.cuda.is_available():
+                    torch.cuda.empty_cache()
+                logger.warning(
+                    "Duplicate pipeline created for key=%s; orphan freed", key
+                )
+            except Exception:  # pragma: no cover
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)